app/routes/api.js

```diff
@@ -13,9 +13,8 @@ const router = new Router();
 
 router.get('/', (req, res) => {
     res.json({
-        message: 'Welcome to the Astro API',
+        message: 'Welcome to the Astro API.',
```

Comment:
We should probably document the api for our own sake and any third-party applications that want to integrate with astro. I documented the old API used in bitcannon a while back [here](https://github.com/aidanharris/bitcannon/wiki/API-V1.0) which could be useful for some inspiration.